lib/ansible/modules/service.py

```diff
@@ -1010,8 +1016,21 @@ def get_service_tools(self):
 
         self.sysrc_cmd = self.module.get_bin_path('sysrc')
 
+    def run_service_cmd(self, status=None):
+        if not status:
+            return (None, '', '')
+
+        cmd = f"{self.svc_cmd} "
+        # Jail name takes precedence over other arguments
+        if self.arguments.startswith(('-j', '-v')):
```

Comment:
Looking at the [man page](https://man.freebsd.org/cgi/man.cgi?service(8)), it appears the options are always supposed to be given first:

> service [-j jail] [-v] [-E var=value] script command

If I'm reading that right, that means we can always use that form and drop the conditional -- at which point we also don't need this function and can just do that inline.

Reply:
Some services take arguments after command as well, for example 

```
service netif enabled lo0
```

some services are -

```
service -j jail_name sendmail enabled
```